sparkmagic/sparkmagic/livyclientlib/command.py

```diff
@@ -44,24 +44,25 @@ def execute(self, session):
             return output
 
     def _get_statement_output(self, session, statement_id):
-        statement_running = True
         out = u""
```

Comment:
The logic of this is confusing to me.  `out` is never assigned to in this file so it is always `u""`.  What purpose does it serve?